GramAddict/plugins/core_arguments.py

```diff
@@ -160,4 +160,18 @@ def __init__(self):
                 "metavar": ("hashtag1", "hashtag2"),
                 "default": None,
             },
+            {
+                "arg": "--scraping-mode",
+                "nargs": None,
+                "help": "generate a file in main directory with given name with target users",
+                "metavar": ("filename"),
+                "default": None,
+            },
+            {
+                "arg": "--total-scrape-limit",
+                "nargs": None,
+                "help": "limit on total scrape per session, 500 by default",
```

Comment:
This should be unneeded. Source limits should be handled by interaction-count. Session limits should be handled by total successful interactions.